### ISSUE TITLE
2 links being active at same time (color of links)

### ISSUE THREAD

**Issue by iraqwarvet31** [fisshy/react-scroll#464]
I'm trying to get each link to have a solid color when active. It works flawlessly until I get to the last link "contact". When I navigate to "contact", it becomes active along with the previous one. So then I have 2 solid color links, when only "contact" should be solid. I took 2 screenshots, click on both links to reference:

https://imgur.com/sedwrZA
https://imgur.com/WI5Ae28

**Comment by fisshy** [comment]
Are both the elements visible in the viewport? 

But please provide a plunkr/codepen to demonstrate the issue and I could take a look at it.

**Comment by iraqwarvet31** [comment]
Yes both elements are visible in the view port. I was able to come up with a hacky solution by increasing the margin on my bottom div, but then I dont get the view I want. Is there a way to force only 1 link being active when both elements are visible?

**Comment by fisshy** [comment]
Maybe you could listen to the events on the elements being activated/inactivated and come up with your own logic to activate only one of them.
